### PR TITLE
Programmes 6537 optimise episode page queries

### PR DESCRIPTION
**Pull request by BjornTwachtmann**

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: composer.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
         "bbc/branding-client": "^v2.6.2",
         "bbc/gel-iconography-assets": "^2.0",
         "bbc/programmes-morph-library": "^1.0",
-        "bbc/programmes-pages-service": "dev-master@dev",
+        "bbc/programmes-pages-service": "dev-PROGRAMMES-6537-optimise-episode-page-queries@dev",
```
Comment (votemike) [inline composer.json]: This needs changing back before merging.

Comment (BjornTwachtmann) [reply composer.json]: Indeed

File: composer.lock [hunk 2]
```diff
@@ -3640,6 +3640,11 @@
         {
             "name": "symfony/lts",
             "version": "v3",
+            "source": {
```
Comment (benolot) [inline composer.lock]: So we don't want to get it from packagist?

Comment (BjornTwachtmann) [reply composer.lock]: Why the fuck did it change that. DAMN YOU COMPOSER

**Pull request merged**